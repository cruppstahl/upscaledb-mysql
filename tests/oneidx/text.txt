DROP TABLE IF EXISTS test;
CREATE TABLE test (file TEXT, id INTEGER PRIMARY KEY) ENGINE=upscaledb;
INSERT INTO test VALUES (LOAD_FILE('/home/ruppc/prj/upscaledb-mysql/tests/blob.txt'), 1);
SELECT * FROM test;
INSERT INTO test VALUES (LOAD_FILE('/home/ruppc/prj/upscaledb-mysql/tests/blob.txt'), 2);
SELECT * FROM test;
