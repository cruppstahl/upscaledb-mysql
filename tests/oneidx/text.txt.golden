--------------
DROP TABLE IF EXISTS test
--------------

--------------
CREATE TABLE test (file TEXT, id INTEGER PRIMARY KEY) ENGINE=upscaledb
--------------

--------------
INSERT INTO test VALUES (LOAD_FILE('/home/ruppc/prj/upscaledb-mysql/tests/blob.txt'), 1)
--------------

--------------
SELECT * FROM test
--------------

file	id
1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890\n	1
--------------
INSERT INTO test VALUES (LOAD_FILE('/home/ruppc/prj/upscaledb-mysql/tests/blob.txt'), 2)
--------------

--------------
SELECT * FROM test
--------------

file	id
1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890\n	1
1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890\n	2
